libs/core/src/lib/nested-list/nested-list.module.ts

```diff
@@ -11,9 +16,14 @@ import { NestedListPopoverComponent } from './nested-list-popover/nested-list-po
 import { PreparedNestedListComponent } from './prepared-nested-list/prepared-nested-list.component';
 import { NestedListStateService } from './nested-list-state.service';
 import { RouterModule } from '@angular/router';
+import { NestedListContentDirective } from './nested-content/nested-list-content.directive';
 
 @NgModule({
-    imports: [CommonModule, PopoverModule, RouterModule],
+    imports: [
+        CommonModule,
+        PopoverModule,
+        RouterModule
```

Comment:
Is `RouterModule` needed in this module?

Reply:
It's needed cause of prepared list built from object